app/components/TagSelector/index.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { blue } from 'style-constants';
+import closeIcon from 'images/closeCircle.svg';
+
+import { Select2 } from 'components/FormFields/SelectField';
```

Comment:
Please give a more descriptive name to the component. `Select2` does not explain the purpose of the component.